backend/.env

```diff
@@ -8,3 +9,7 @@ EMAIL_HOST=smtp.gmail.com
 CLUSTER=no
 JWT_RESET_PASSWORD_EXPIRES_IN=1h
 LOCAL_DEV_ENV=http://localhost:3500/
+
+
+EMAIL_PASS_FOR_NODEMAILER=<enter your gmail pass key>
```

Comment:
can't we reuse `EMAIL_USER` and `EMAIL_PASS` as those 2 are the community email and the email's password respectively

Reply:
yes, we can use EMAIL_USER and EMAIL_PASS